addons/gsri_opex/functions/fremm/fn_initFremm.sqf

```diff
@@ -42,9 +42,22 @@ if(isServer) then {
 	([_ship, 'Land_Destroyer_01_hull_05_F'] call bis_fnc_destroyer01GetShipPart) setObjectTextureGlobal [0, (_ship getVariable "GSRI_FREMM_nameplate")];
 
 	// Add map marker
-	private _mk = createMarker [format["marker_destroyer_%1", _ship getVariable "GSRI_FREMM_shipIndex"], _ship];
-	_mk setMarkerType "flag_France";
-	_mk setMarkerText (_ship getVariable "GSRI_FREMM_fullname");
+	private _fullname = _ship getVariable ["GSRI_FREMM_fullname", ""];
+	private _index = _ship getVariable "GSRI_FREMM_shipIndex";
```

Comment:
that is much inline nesting (refactoring not mandatory)

Reply:
I don't get it, what's the problem with retrieving these values clientside ?

Reply:
Sorry, the comment was ment for the next lines.
The problem is the inlining of instructions.
It would be easier to read with intermediary instructions.